tesseract/messaging/evm/src/gas_oracle.rs

```diff
@@ -310,6 +195,44 @@ fn get_cost_of_one_wei(eth_usd: U256) -> U256 {
 	eth_usd / eth_to_wei
 }
 
+async fn get_price_from_uniswap_router(
+	ismp_host: H160,
+	client: Arc<Provider<Http>>,
+) -> Result<U256, Error> {
+	let host = EvmHost::new(ismp_host, client.clone());
+	let params = host.host_params().call().await?;
```

Comment:
Check if we are on testnet by checking the hyperbridge state machine ID on the ISMP host and return a static value for the native token price.